activesupport/lib/active_support/duration.rb

```diff
@@ -86,7 +93,7 @@ def eql?(other)
       Duration === other && other.value.eql?(value)
     end
 
-    def hash
+    def hash # :nodoc:
```

Comment:
Yes, I know. If we nodoc it will Rdoc then point to the Object#hash documentation?